core/src/main/java/io/grpc/CallCredentials.java

```diff
@@ -65,6 +72,7 @@
    * @param applier The outlet of the produced headers. It can be called either before or after this
    *        method returns.
    */
+  @ExperimentalApi("https//github.com/grpc/grpc-java/issues/1914")
   void applyRequestMetadata(
```

Comment:
This is not a valid.  You can't make part of an interface experimental.   All implementers have to implement it, which will break if it is ever removed.